src/app/containers/ATIAnalytics/atiUrl/index.js

```diff
@@ -141,13 +144,75 @@ const atiPageViewParams = ({
     },
   ];
 
-  const cleanedValues = pageViewBeaconValues.filter(({ value }) => value);
+  return getAtiUrl(pageViewBeaconValues);
+};
 
-  const parsedAtiValues = cleanedValues.map(({ key, value, wrap }) =>
-    wrap ? `${key}=[${value}]` : `${key}=${value}`,
-  );
+export const buildATIEventTrackUrl = ({
```

Comment:
This needs tests

Reply:
Done!

https://github.com/bbc/simorgh/pull/3559/files#diff-1ce23abdc9b1adb0035631ba37076927R115-R132